client/Assets/Scripts/Battle/BattleManager.cs

```diff
@@ -276,4 +288,20 @@ private List<UIReward> CreateRewardsList()
 		}
 		return rewards;
 	}
+
+	private void StatOverride(Protobuf.Messages.Action action)
+	{
+		BattleUnit target = battleUnitsUI.Single(unit => unit.SelectedUnit.id == action.StatOverride.TargetId);
```

Comment:
```suggestion
						BattleUnit target = battleUnitsUI.Single(unit => unit.SelectedUnit != null && unit.SelectedUnit.id == action.StatOverride.TargetId);
```

Sorry, the same problem arrises here, it's the last one, I've checked.

Reply:
Nice catch!